ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -1792,7 +1794,7 @@ def start_sniffer(self, pcap_path, tcpdump_filter, timeout):
                 wait_timer.cancel()
             # Return code here could be 0, so we need to explicitly check for None
             if process.returncode is not None:
-                self.log("Tcpdump process {} terminated".format(process.args))
+                self.log("Tcpdump process {} terminated".format(' '.join(process.command)))
```

Comment:
Is there a reason that `process.command` is needed here? Can we not just print ` command` var?

Reply:
We want to store and print command for each process in process_list